### ISSUE TITLE
createLeafletElement() must be implemented     at HeatmapLayer.createLeafletElement (MapLayer.js:48)

### ISSUE THREAD

**Issue by xinghengwang** [OpenGov/react-leaflet-heatmap-layer#13]
See this issue: 
https://github.com/PaulLeCam/react-leaflet/issues/275

**Comment by jetaix** [comment]
Same issue :/ Need help please

**Comment by hughlomas** [comment]
@jetaix As a temporary fix, you can roll back to a version prior to 1.1.0 (such as 1.0.3) until this issue is addressed.

**Comment by timfpark** [comment]
You can work around this by adding this line:

      HeatmapLayer.prototype.createLeafletElement = function() {};

in node_modules/react-leaflet-heatmap-layer/lib/HeatmapLayer.js after this function:

  function HeatmapLayer() {
    _classCallCheck(this, HeatmapLayer);

    return _possibleConstructorReturn(this, _MapLayer.apply(this, arguments));
  }

**Comment by morkro** [comment]
After implementing @timfpark's fix, I get this error:

```
Uncaught TypeError: Cannot read property 'getSize' of undefined
```

Are there any plans to fix this?

**Comment by evansiroky** [comment]
FYI, there is an open PR (#10) to upgrade to leaflet 1.0

**Comment by caseysiebel** [comment]
I'm still experiencing this issue, has this PR been merged yet?

**Comment by MartinSahlen** [comment]
@caseysiebel No, we are still waiting for it to get in.

**Comment by skipjack** [comment]
I think this can be closed as #10 was merged  🎉
